lib/passport-configurator.js

```diff
@@ -405,6 +406,26 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
         }
       );
       break;
+    case 'fb-token':
```

Comment:
To complete my answer, all the `passport-facebook-token` strategy does is call Facebook with the token and get the user's profile information, confirming it and validating it.  
As you may be aware, the `passport-oauth2` strategy does much more.